source/_layouts/catalog.html

```diff
@@ -3,108 +3,81 @@
-                <button class="filter" data-filter='.{{ item }}'>{{ item | capitalize }}</button>
-                {% capture tempList %}{{ typeList }}{{ item }},{% endcapture %}
-                {% assign typeList = tempList %}
-              {% endif %}
-            {% endfor %}
-            {% endif %}
-          {% endfor %}
-        </fieldset>
-        <fieldset class="filter-set">
-          <span class="filter-cat callout">Size</span>
-           {% assign typeList = ',' %}
-          {% for product in site.categories.products %}
-            {% if product.catalog == page.slug %}
-            {% for item in product.size %}
-              {% capture compareType %},{{ item }},{% endcapture %}
-              {% if typeList contains compareType %}{% else %}
-                <button class="filter" data-filter='.{{ item }}'>{{ item | capitalize }}</button>
-                {% capture tempList %}{{ typeList }}{{ item }},{% endcapture %}
-                {% assign typeList = tempList %}
-              {% endif %}
-            {% endfor %}
-            {% endif %}
-          {% endfor %}
-        </fieldset>
-        <fieldset class="filter-set">
-          <span class="filter-cat callout">Color</span>
-           {% assign typeList = ',' %}
-          {% for product in site.categories.products %}
-            {% if product.catalog == page.slug %}
-            {% for item in product.color %}
-              {% capture compareType %},{{ item }},{% endcapture %}
-              {% if typeList contains compareType %}{% else %}
-                <button class="filter" data-filter='.{{ item }}'>{{ item | capitalize }}</button>
-                {% capture tempList %}{{ typeList }}{{ item }},{% endcapture %}
-                {% assign typeList = tempList %}
-              {% endif %}
-            {% endfor %}
-            {% endif %}
-          {% endfor %}
-        </fieldset>
-      </form>
-    </aside>
-
-    <ul id="filter-container" class="grid medium">
-      {% for product in site.categories.products %}
-      {% if product.catalog == page.slug %}
-        {% include product-tile.html post=product filter=true %}
-      {% endif %}
-      {% endfor %}
-    </ul>
+<section>
+  {%- comment -%} if there's a banner image... {%- endcomment -%}
+  {%- if page.cover -%}
+    {%- include billboard.html 
+      image='https://placehold.it/1200x600?text=Banner'
+      title=page.title
+      button_label='<strong>Download a PDF</strong> of this catalog'
+      button_url=page.pdf
+    -%}
+  {%- else -%}
+    {%- comment -%} otherwise, just show the title {%- endcomment -%}
+    <div class="padding-x-outside">
+      <div data-ui-wrapper="wide" class="margin-top-header">
+        <h1 class="t-align-center">{{ page.title }}</h1>
+        {%- if page.pdf -%}
+          <div class="padding-top t-align-center">
+            {%- include button_link.html
+              label='<strong>Download a PDF</strong> of this catalog'
```

Comment:
We should make this target="_blank"

Reply:
oh good call.

later, I think I'd like to dress this button up a bit more -- maybe it displays a download icon or even a thumb of the PDF cover. but will depend a little on whether we end up keeping the banner graphic behind it.